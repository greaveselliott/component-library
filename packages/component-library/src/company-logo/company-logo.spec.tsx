import React from 'react';
import { shallow } from 'enzyme';

import CompanyLogo from './company-logo.component';

test('Renders correctly', () =>
  expect(
    shallow(<CompanyLogo href="/" title="A descriptive title" />),
  ).toMatchSnapshot());
