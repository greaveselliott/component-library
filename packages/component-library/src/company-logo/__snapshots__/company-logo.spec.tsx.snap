// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<a
  className="company-logo"
  href="/"
  title="A descriptive title"
>
  <img
    alt="The company logo"
    className="company-logo__image"
    src={[Function]}
  />
</a>
`;
